src/index.ts

```diff
@@ -101,6 +103,26 @@ async function StartApp() {
     next()
   })
 
+  app.get('/sse/:token', async (req, res) => {
+    const token = req.params.token
+
+    if (token) {
+      const session = await createSession(req, res)
+      const notificationSSEService = new NotificationSSEService(APP_SOURCE)
+
+      setInterval(async () => {
+        const notifications = await notificationSSEService.getAllUser({
+          notificationToken: token,
+        })
+
+        if (notifications?.length > 0) {
+          session.push(notifications)
+        }
+        // 5 secondes
+      }, 5000)
```

Comment:
You may create a channel for each user (stored in memory)

and broadcast to each user , 

Examples: https://github.com/MatthewWid/better-sse/blob/master/examples/resource-monitor/channels/resource.ts#L8